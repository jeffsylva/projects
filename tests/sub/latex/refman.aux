\relax 
\ifx\hyper@anchor\@undefined
\global \let \oldcontentsline\contentsline
\gdef \contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global \let \oldnewlabel\newlabel
\gdef \newlabel#1#2{\newlabelxx{#1}#2}
\gdef \newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\let \contentsline\oldcontentsline
\let \newlabel\oldnewlabel}
\else
\global \let \hyper@last\relax 
\fi

\@writefile{toc}{\contentsline {chapter}{\numberline {1}Bug List}{1}{chapter.1}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newlabel{bug}{{1}{1}{Bug List\relax }{chapter.1}{}}
\newlabel{bug__bug000001}{{1}{1}{Bug List\relax }{chapter.1}{}}
\newlabel{bug__bug000002}{{1}{1}{Bug List\relax }{chapter.1}{}}
\newlabel{bug__bug000003}{{1}{1}{Bug List\relax }{chapter.1}{}}
\newlabel{bug__bug000004}{{1}{1}{Bug List\relax }{chapter.1}{}}
\@writefile{toc}{\contentsline {chapter}{\numberline {2}Class Index}{3}{chapter.2}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {section}{\numberline {2.1}Class List}{3}{section.2.1}}
\@writefile{toc}{\contentsline {chapter}{\numberline {3}File Index}{5}{chapter.3}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {section}{\numberline {3.1}File List}{5}{section.3.1}}
\@writefile{toc}{\contentsline {chapter}{\numberline {4}Class Documentation}{7}{chapter.4}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {section}{\numberline {4.1}CollisionTests Class Reference}{7}{section.4.1}}
\newlabel{classCollisionTests}{{4.1}{7}{CollisionTests Class Reference\relax }{section.4.1}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {4.1.1}Member Function Documentation}{8}{subsection.4.1.1}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {4.1.1.1}bulletcollsiontest}{8}{subsubsection.4.1.1.1}}
\newlabel{classCollisionTests_a5e00b872973bc79d09ee1dbc47550ab1}{{4.1.1.1}{8}{bulletcollsiontest\relax }{subsubsection.4.1.1.1}{}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {4.1.1.2}bulletcollsiontest1}{8}{subsubsection.4.1.1.2}}
\newlabel{classCollisionTests_a2e58eacdce778bf4cb996b5cc0388c3f}{{4.1.1.2}{8}{bulletcollsiontest1\relax }{subsubsection.4.1.1.2}{}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {4.1.1.3}collisiontest}{8}{subsubsection.4.1.1.3}}
\newlabel{classCollisionTests_af5e85370e1846385a61da97eeb7e8b28}{{4.1.1.3}{8}{collisiontest\relax }{subsubsection.4.1.1.3}{}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {4.1.1.4}collisiontest1}{8}{subsubsection.4.1.1.4}}
\newlabel{classCollisionTests_a787429e5b0361daacf1df123560ca7c5}{{4.1.1.4}{8}{collisiontest1\relax }{subsubsection.4.1.1.4}{}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {4.1.1.5}CPPUNIT\_\discretionary {-}{}{}TEST}{8}{subsubsection.4.1.1.5}}
\newlabel{classCollisionTests_a71fe8a9d6ecb756ffc41dc8cce239944}{{4.1.1.5}{8}{CPPUNIT\_\-TEST\relax }{subsubsection.4.1.1.5}{}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {4.1.1.6}CPPUNIT\_\discretionary {-}{}{}TEST}{8}{subsubsection.4.1.1.6}}
\newlabel{classCollisionTests_a516a73cfbac5616bd64518c97b922fb8}{{4.1.1.6}{8}{CPPUNIT\_\-TEST\relax }{subsubsection.4.1.1.6}{}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {4.1.1.7}CPPUNIT\_\discretionary {-}{}{}TEST}{8}{subsubsection.4.1.1.7}}
\newlabel{classCollisionTests_aa582a183125d00737483f6ecf9ba1e81}{{4.1.1.7}{8}{CPPUNIT\_\-TEST\relax }{subsubsection.4.1.1.7}{}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {4.1.1.8}CPPUNIT\_\discretionary {-}{}{}TEST}{8}{subsubsection.4.1.1.8}}
\newlabel{classCollisionTests_a6f2e114f6c020b7650a9357386a85084}{{4.1.1.8}{8}{CPPUNIT\_\-TEST\relax }{subsubsection.4.1.1.8}{}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {4.1.1.9}CPPUNIT\_\discretionary {-}{}{}TEST\_\discretionary {-}{}{}SUITE}{8}{subsubsection.4.1.1.9}}
\newlabel{classCollisionTests_a542155fcf2acb7a46506e5924f32de86}{{4.1.1.9}{8}{CPPUNIT\_\-TEST\_\-SUITE\relax }{subsubsection.4.1.1.9}{}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {4.1.1.10}CPPUNIT\_\discretionary {-}{}{}TEST\_\discretionary {-}{}{}SUITE\_\discretionary {-}{}{}END}{8}{subsubsection.4.1.1.10}}
\newlabel{classCollisionTests_ad9399d823f41c3dcf60d83b3d7b96574}{{4.1.1.10}{8}{CPPUNIT\_\-TEST\_\-SUITE\_\-END\relax }{subsubsection.4.1.1.10}{}}
\@writefile{toc}{\contentsline {chapter}{\numberline {5}File Documentation}{9}{chapter.5}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {section}{\numberline {5.1}AllegroTest.cc File Reference}{9}{section.5.1}}
\newlabel{AllegroTest_8cc}{{5.1}{9}{AllegroTest.cc File Reference\relax }{section.5.1}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {5.1.1}Detailed Description}{9}{subsection.5.1.1}}
\@writefile{toc}{\contentsline {subsection}{\numberline {5.1.2}Function Documentation}{9}{subsection.5.1.2}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {5.1.2.1}collision}{9}{subsubsection.5.1.2.1}}
\newlabel{AllegroTest_8cc_a06097efd0922a39bda31d17264b6ec68}{{5.1.2.1}{9}{collision\relax }{subsubsection.5.1.2.1}{}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {5.1.2.2}collision1}{9}{subsubsection.5.1.2.2}}
\newlabel{AllegroTest_8cc_afec33bb30fc1329269f9fd64745a59d3}{{5.1.2.2}{9}{collision1\relax }{subsubsection.5.1.2.2}{}}
\@input{AllegroTest_8h.aux}
\@input{collisiontests_8h.aux}
\@input{mainTest_8cc.aux}
