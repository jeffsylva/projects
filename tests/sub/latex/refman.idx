\indexentry{CollisionTests@{CollisionTests}|hyperpage}{7}
\indexentry{CollisionTests@{CollisionTests}!bulletcollsiontest@{bulletcollsiontest}|hyperpage}{8}
\indexentry{bulletcollsiontest@{bulletcollsiontest}!CollisionTests@{CollisionTests}|hyperpage}{8}
\indexentry{CollisionTests@{CollisionTests}!bulletcollsiontest1@{bulletcollsiontest1}|hyperpage}{8}
\indexentry{bulletcollsiontest1@{bulletcollsiontest1}!CollisionTests@{CollisionTests}|hyperpage}{8}
\indexentry{CollisionTests@{CollisionTests}!collisiontest@{collisiontest}|hyperpage}{8}
\indexentry{collisiontest@{collisiontest}!CollisionTests@{CollisionTests}|hyperpage}{8}
\indexentry{CollisionTests@{CollisionTests}!collisiontest1@{collisiontest1}|hyperpage}{8}
\indexentry{collisiontest1@{collisiontest1}!CollisionTests@{CollisionTests}|hyperpage}{8}
\indexentry{CollisionTests@{CollisionTests}!CPPUNIT\_\discretionary {-}{}{}TEST@{CPPUNIT\_\discretionary {-}{}{}TEST}|hyperpage}{8}
\indexentry{CPPUNIT\_\discretionary {-}{}{}TEST@{CPPUNIT\_\discretionary {-}{}{}TEST}!CollisionTests@{CollisionTests}|hyperpage}{8}
\indexentry{CollisionTests@{CollisionTests}!CPPUNIT\_\discretionary {-}{}{}TEST@{CPPUNIT\_\discretionary {-}{}{}TEST}|hyperpage}{8}
\indexentry{CPPUNIT\_\discretionary {-}{}{}TEST@{CPPUNIT\_\discretionary {-}{}{}TEST}!CollisionTests@{CollisionTests}|hyperpage}{8}
\indexentry{CollisionTests@{CollisionTests}!CPPUNIT\_\discretionary {-}{}{}TEST@{CPPUNIT\_\discretionary {-}{}{}TEST}|hyperpage}{8}
\indexentry{CPPUNIT\_\discretionary {-}{}{}TEST@{CPPUNIT\_\discretionary {-}{}{}TEST}!CollisionTests@{CollisionTests}|hyperpage}{8}
\indexentry{CollisionTests@{CollisionTests}!CPPUNIT\_\discretionary {-}{}{}TEST@{CPPUNIT\_\discretionary {-}{}{}TEST}|hyperpage}{8}
\indexentry{CPPUNIT\_\discretionary {-}{}{}TEST@{CPPUNIT\_\discretionary {-}{}{}TEST}!CollisionTests@{CollisionTests}|hyperpage}{8}
\indexentry{CollisionTests@{CollisionTests}!CPPUNIT\_\discretionary {-}{}{}TEST\_\discretionary {-}{}{}SUITE@{CPPUNIT\_\discretionary {-}{}{}TEST\_\discretionary {-}{}{}SUITE}|hyperpage}{8}
\indexentry{CPPUNIT\_\discretionary {-}{}{}TEST\_\discretionary {-}{}{}SUITE@{CPPUNIT\_\discretionary {-}{}{}TEST\_\discretionary {-}{}{}SUITE}!CollisionTests@{CollisionTests}|hyperpage}{8}
\indexentry{CollisionTests@{CollisionTests}!CPPUNIT\_\discretionary {-}{}{}TEST\_\discretionary {-}{}{}SUITE\_\discretionary {-}{}{}END@{CPPUNIT\_\discretionary {-}{}{}TEST\_\discretionary {-}{}{}SUITE\_\discretionary {-}{}{}END}|hyperpage}{8}
\indexentry{CPPUNIT\_\discretionary {-}{}{}TEST\_\discretionary {-}{}{}SUITE\_\discretionary {-}{}{}END@{CPPUNIT\_\discretionary {-}{}{}TEST\_\discretionary {-}{}{}SUITE\_\discretionary {-}{}{}END}!CollisionTests@{CollisionTests}|hyperpage}{8}
\indexentry{AllegroTest.cc@{AllegroTest.cc}|hyperpage}{9}
\indexentry{AllegroTest.cc@{AllegroTest.cc}!collision@{collision}|hyperpage}{9}
\indexentry{collision@{collision}!AllegroTest.cc@{AllegroTest.cc}|hyperpage}{9}
\indexentry{AllegroTest.cc@{AllegroTest.cc}!collision1@{collision1}|hyperpage}{9}
\indexentry{collision1@{collision1}!AllegroTest.cc@{AllegroTest.cc}|hyperpage}{9}
\indexentry{AllegroTest.h@{AllegroTest.h}|hyperpage}{10}
\indexentry{AllegroTest.h@{AllegroTest.h}!collision@{collision}|hyperpage}{10}
\indexentry{collision@{collision}!AllegroTest.h@{AllegroTest.h}|hyperpage}{10}
\indexentry{AllegroTest.h@{AllegroTest.h}!collision1@{collision1}|hyperpage}{10}
\indexentry{collision1@{collision1}!AllegroTest.h@{AllegroTest.h}|hyperpage}{10}
\indexentry{collisiontests.h@{collisiontests.h}|hyperpage}{11}
\indexentry{mainTest.cc@{mainTest.cc}|hyperpage}{12}
\indexentry{mainTest.cc@{mainTest.cc}!main@{main}|hyperpage}{12}
\indexentry{main@{main}!mainTest.cc@{mainTest.cc}|hyperpage}{12}
