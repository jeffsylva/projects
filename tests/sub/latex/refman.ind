\begin{theindex}

  \item {AllegroTest.cc}, \hyperpage{9}
    \subitem {collision}, \hyperpage{9}
    \subitem {collision1}, \hyperpage{9}
  \item {AllegroTest.h}, \hyperpage{10}
    \subitem {collision}, \hyperpage{10}
    \subitem {collision1}, \hyperpage{10}

  \indexspace

  \item {bulletcollsiontest}
    \subitem {CollisionTests}, \hyperpage{8}
  \item {bulletcollsiontest1}
    \subitem {CollisionTests}, \hyperpage{8}

  \indexspace

  \item {collision}
    \subitem {AllegroTest.cc}, \hyperpage{9}
    \subitem {AllegroTest.h}, \hyperpage{10}
  \item {collision1}
    \subitem {AllegroTest.cc}, \hyperpage{9}
    \subitem {AllegroTest.h}, \hyperpage{10}
  \item {collisiontest}
    \subitem {CollisionTests}, \hyperpage{8}
  \item {collisiontest1}
    \subitem {CollisionTests}, \hyperpage{8}
  \item {CollisionTests}, \hyperpage{7}
    \subitem {bulletcollsiontest}, \hyperpage{8}
    \subitem {bulletcollsiontest1}, \hyperpage{8}
    \subitem {collisiontest}, \hyperpage{8}
    \subitem {collisiontest1}, \hyperpage{8}
    \subitem {CPPUNIT\_\discretionary {-}{}{}TEST}, \hyperpage{8}
    \subitem {CPPUNIT\_\discretionary {-}{}{}TEST\_\discretionary {-}{}{}SUITE}, \hyperpage{8}
    \subitem {CPPUNIT\_\discretionary {-}{}{}TEST\_\discretionary {-}{}{}SUITE\_\discretionary {-}{}{}END}, \hyperpage{8}
  \item {collisiontests.h}, \hyperpage{11}
  \item {CPPUNIT\_\discretionary {-}{}{}TEST}
    \subitem {CollisionTests}, \hyperpage{8}
  \item {CPPUNIT\_\discretionary {-}{}{}TEST\_\discretionary {-}{}{}SUITE}
    \subitem {CollisionTests}, \hyperpage{8}
  \item {CPPUNIT\_\discretionary {-}{}{}TEST\_\discretionary {-}{}{}SUITE\_\discretionary {-}{}{}END}
    \subitem {CollisionTests}, \hyperpage{8}

  \indexspace

  \item {main}
    \subitem {mainTest.cc}, \hyperpage{12}
  \item {mainTest.cc}, \hyperpage{12}
    \subitem {main}, \hyperpage{12}

\end{theindex}
