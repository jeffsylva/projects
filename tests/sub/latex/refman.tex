\documentclass[a4paper]{book}
\usepackage{a4wide}
\usepackage{makeidx}
\usepackage{graphicx}
\usepackage{multicol}
\usepackage{float}
\usepackage{listings}
\usepackage{color}
\usepackage{textcomp}
\usepackage{alltt}
\usepackage{times}
\usepackage{ifpdf}
\ifpdf
\usepackage[pdftex,
            pagebackref=true,
            colorlinks=true,
            linkcolor=blue,
            unicode
           ]{hyperref}
\else
\usepackage[ps2pdf,
            pagebackref=true,
            colorlinks=true,
            linkcolor=blue,
            unicode
           ]{hyperref}
\usepackage{pspicture}
\fi
\usepackage[utf8]{inputenc}
\usepackage{doxygen}
\lstset{language=C++,inputencoding=utf8,basicstyle=\footnotesize,breaklines=true,breakatwhitespace=true,tabsize=8,numbers=left }
\makeindex
\setcounter{tocdepth}{3}
\renewcommand{\footrulewidth}{0.4pt}
\begin{document}
\hypersetup{pageanchor=false}
\begin{titlepage}
\vspace*{7cm}
\begin{center}
{\Large DELTAFORCETESTDRIVENDEVELOPMENT \\[1ex]\large CPSC2720PROJECT }\\
\vspace*{1cm}
{\large Generated by Doxygen 1.6.1}\\
\vspace*{0.5cm}
{\small Mon Apr 18 23:47:40 2016}\\
\end{center}
\end{titlepage}
\clearemptydoublepage
\pagenumbering{roman}
\tableofcontents
\clearemptydoublepage
\pagenumbering{arabic}
\hypersetup{pageanchor=true}
\chapter{Bug List}
\label{bug}
\hypertarget{bug}{}
\input{bug}
\chapter{Class Index}
\input{annotated}
\chapter{File Index}
\input{files}
\chapter{Class Documentation}
\input{classCollisionTests}
\chapter{File Documentation}
\input{AllegroTest_8cc}
\include{AllegroTest_8h}
\include{collisiontests_8h}
\include{mainTest_8cc}
\printindex
\end{document}
