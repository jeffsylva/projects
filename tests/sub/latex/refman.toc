\contentsline {chapter}{\numberline {1}Bug List}{1}{chapter.1}
\contentsline {chapter}{\numberline {2}Class Index}{3}{chapter.2}
\contentsline {section}{\numberline {2.1}Class List}{3}{section.2.1}
\contentsline {chapter}{\numberline {3}File Index}{5}{chapter.3}
\contentsline {section}{\numberline {3.1}File List}{5}{section.3.1}
\contentsline {chapter}{\numberline {4}Class Documentation}{7}{chapter.4}
\contentsline {section}{\numberline {4.1}CollisionTests Class Reference}{7}{section.4.1}
\contentsline {subsection}{\numberline {4.1.1}Member Function Documentation}{8}{subsection.4.1.1}
\contentsline {subsubsection}{\numberline {4.1.1.1}bulletcollsiontest}{8}{subsubsection.4.1.1.1}
\contentsline {subsubsection}{\numberline {4.1.1.2}bulletcollsiontest1}{8}{subsubsection.4.1.1.2}
\contentsline {subsubsection}{\numberline {4.1.1.3}collisiontest}{8}{subsubsection.4.1.1.3}
\contentsline {subsubsection}{\numberline {4.1.1.4}collisiontest1}{8}{subsubsection.4.1.1.4}
\contentsline {subsubsection}{\numberline {4.1.1.5}CPPUNIT\_\discretionary {-}{}{}TEST}{8}{subsubsection.4.1.1.5}
\contentsline {subsubsection}{\numberline {4.1.1.6}CPPUNIT\_\discretionary {-}{}{}TEST}{8}{subsubsection.4.1.1.6}
\contentsline {subsubsection}{\numberline {4.1.1.7}CPPUNIT\_\discretionary {-}{}{}TEST}{8}{subsubsection.4.1.1.7}
\contentsline {subsubsection}{\numberline {4.1.1.8}CPPUNIT\_\discretionary {-}{}{}TEST}{8}{subsubsection.4.1.1.8}
\contentsline {subsubsection}{\numberline {4.1.1.9}CPPUNIT\_\discretionary {-}{}{}TEST\_\discretionary {-}{}{}SUITE}{8}{subsubsection.4.1.1.9}
\contentsline {subsubsection}{\numberline {4.1.1.10}CPPUNIT\_\discretionary {-}{}{}TEST\_\discretionary {-}{}{}SUITE\_\discretionary {-}{}{}END}{8}{subsubsection.4.1.1.10}
\contentsline {chapter}{\numberline {5}File Documentation}{9}{chapter.5}
\contentsline {section}{\numberline {5.1}AllegroTest.cc File Reference}{9}{section.5.1}
\contentsline {subsection}{\numberline {5.1.1}Detailed Description}{9}{subsection.5.1.1}
\contentsline {subsection}{\numberline {5.1.2}Function Documentation}{9}{subsection.5.1.2}
\contentsline {subsubsection}{\numberline {5.1.2.1}collision}{9}{subsubsection.5.1.2.1}
\contentsline {subsubsection}{\numberline {5.1.2.2}collision1}{9}{subsubsection.5.1.2.2}
\contentsline {section}{\numberline {5.2}AllegroTest.h File Reference}{10}{section.5.2}
\contentsline {subsection}{\numberline {5.2.1}Detailed Description}{10}{subsection.5.2.1}
\contentsline {subsection}{\numberline {5.2.2}Function Documentation}{10}{subsection.5.2.2}
\contentsline {subsubsection}{\numberline {5.2.2.1}collision}{10}{subsubsection.5.2.2.1}
\contentsline {subsubsection}{\numberline {5.2.2.2}collision1}{10}{subsubsection.5.2.2.2}
\contentsline {section}{\numberline {5.3}collisiontests.h File Reference}{11}{section.5.3}
\contentsline {subsection}{\numberline {5.3.1}Detailed Description}{11}{subsection.5.3.1}
\contentsline {section}{\numberline {5.4}mainTest.cc File Reference}{12}{section.5.4}
\contentsline {subsection}{\numberline {5.4.1}Detailed Description}{12}{subsection.5.4.1}
\contentsline {subsection}{\numberline {5.4.2}Function Documentation}{12}{subsection.5.4.2}
\contentsline {subsubsection}{\numberline {5.4.2.1}main}{12}{subsubsection.5.4.2.1}
