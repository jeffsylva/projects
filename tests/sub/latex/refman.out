\BOOKMARK [0][-]{chapter.1}{\376\377\000B\000u\000g\000\040\000L\000i\000s\000t}{}
\BOOKMARK [0][-]{chapter.2}{\376\377\000C\000l\000a\000s\000s\000\040\000I\000n\000d\000e\000x}{}
\BOOKMARK [1][-]{section.2.1}{\376\377\000C\000l\000a\000s\000s\000\040\000L\000i\000s\000t}{chapter.2}
\BOOKMARK [0][-]{chapter.3}{\376\377\000F\000i\000l\000e\000\040\000I\000n\000d\000e\000x}{}
\BOOKMARK [1][-]{section.3.1}{\376\377\000F\000i\000l\000e\000\040\000L\000i\000s\000t}{chapter.3}
\BOOKMARK [0][-]{chapter.4}{\376\377\000C\000l\000a\000s\000s\000\040\000D\000o\000c\000u\000m\000e\000n\000t\000a\000t\000i\000o\000n}{}
\BOOKMARK [1][-]{section.4.1}{\376\377\000C\000o\000l\000l\000i\000s\000i\000o\000n\000T\000e\000s\000t\000s\000\040\000C\000l\000a\000s\000s\000\040\000R\000e\000f\000e\000r\000e\000n\000c\000e}{chapter.4}
\BOOKMARK [2][-]{subsection.4.1.1}{\376\377\000M\000e\000m\000b\000e\000r\000\040\000F\000u\000n\000c\000t\000i\000o\000n\000\040\000D\000o\000c\000u\000m\000e\000n\000t\000a\000t\000i\000o\000n}{section.4.1}
\BOOKMARK [3][-]{subsubsection.4.1.1.1}{\376\377\000b\000u\000l\000l\000e\000t\000c\000o\000l\000l\000s\000i\000o\000n\000t\000e\000s\000t}{subsection.4.1.1}
\BOOKMARK [3][-]{subsubsection.4.1.1.2}{\376\377\000b\000u\000l\000l\000e\000t\000c\000o\000l\000l\000s\000i\000o\000n\000t\000e\000s\000t\0001}{subsection.4.1.1}
\BOOKMARK [3][-]{subsubsection.4.1.1.3}{\376\377\000c\000o\000l\000l\000i\000s\000i\000o\000n\000t\000e\000s\000t}{subsection.4.1.1}
\BOOKMARK [3][-]{subsubsection.4.1.1.4}{\376\377\000c\000o\000l\000l\000i\000s\000i\000o\000n\000t\000e\000s\000t\0001}{subsection.4.1.1}
\BOOKMARK [3][-]{subsubsection.4.1.1.5}{\376\377\000C\000P\000P\000U\000N\000I\000T\000\137\000T\000E\000S\000T}{subsection.4.1.1}
\BOOKMARK [3][-]{subsubsection.4.1.1.6}{\376\377\000C\000P\000P\000U\000N\000I\000T\000\137\000T\000E\000S\000T}{subsection.4.1.1}
\BOOKMARK [3][-]{subsubsection.4.1.1.7}{\376\377\000C\000P\000P\000U\000N\000I\000T\000\137\000T\000E\000S\000T}{subsection.4.1.1}
\BOOKMARK [3][-]{subsubsection.4.1.1.8}{\376\377\000C\000P\000P\000U\000N\000I\000T\000\137\000T\000E\000S\000T}{subsection.4.1.1}
\BOOKMARK [3][-]{subsubsection.4.1.1.9}{\376\377\000C\000P\000P\000U\000N\000I\000T\000\137\000T\000E\000S\000T\000\137\000S\000U\000I\000T\000E}{subsection.4.1.1}
\BOOKMARK [3][-]{subsubsection.4.1.1.10}{\376\377\000C\000P\000P\000U\000N\000I\000T\000\137\000T\000E\000S\000T\000\137\000S\000U\000I\000T\000E\000\137\000E\000N\000D}{subsection.4.1.1}
\BOOKMARK [0][-]{chapter.5}{\376\377\000F\000i\000l\000e\000\040\000D\000o\000c\000u\000m\000e\000n\000t\000a\000t\000i\000o\000n}{}
\BOOKMARK [1][-]{section.5.1}{\376\377\000A\000l\000l\000e\000g\000r\000o\000T\000e\000s\000t\000.\000c\000c\000\040\000F\000i\000l\000e\000\040\000R\000e\000f\000e\000r\000e\000n\000c\000e}{chapter.5}
\BOOKMARK [2][-]{subsection.5.1.1}{\376\377\000D\000e\000t\000a\000i\000l\000e\000d\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n}{section.5.1}
\BOOKMARK [2][-]{subsection.5.1.2}{\376\377\000F\000u\000n\000c\000t\000i\000o\000n\000\040\000D\000o\000c\000u\000m\000e\000n\000t\000a\000t\000i\000o\000n}{section.5.1}
\BOOKMARK [3][-]{subsubsection.5.1.2.1}{\376\377\000c\000o\000l\000l\000i\000s\000i\000o\000n}{subsection.5.1.2}
\BOOKMARK [3][-]{subsubsection.5.1.2.2}{\376\377\000c\000o\000l\000l\000i\000s\000i\000o\000n\0001}{subsection.5.1.2}
\BOOKMARK [1][-]{section.5.2}{\376\377\000A\000l\000l\000e\000g\000r\000o\000T\000e\000s\000t\000.\000h\000\040\000F\000i\000l\000e\000\040\000R\000e\000f\000e\000r\000e\000n\000c\000e}{chapter.5}
\BOOKMARK [2][-]{subsection.5.2.1}{\376\377\000D\000e\000t\000a\000i\000l\000e\000d\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n}{section.5.2}
\BOOKMARK [2][-]{subsection.5.2.2}{\376\377\000F\000u\000n\000c\000t\000i\000o\000n\000\040\000D\000o\000c\000u\000m\000e\000n\000t\000a\000t\000i\000o\000n}{section.5.2}
\BOOKMARK [3][-]{subsubsection.5.2.2.1}{\376\377\000c\000o\000l\000l\000i\000s\000i\000o\000n}{subsection.5.2.2}
\BOOKMARK [3][-]{subsubsection.5.2.2.2}{\376\377\000c\000o\000l\000l\000i\000s\000i\000o\000n\0001}{subsection.5.2.2}
\BOOKMARK [1][-]{section.5.3}{\376\377\000c\000o\000l\000l\000i\000s\000i\000o\000n\000t\000e\000s\000t\000s\000.\000h\000\040\000F\000i\000l\000e\000\040\000R\000e\000f\000e\000r\000e\000n\000c\000e}{chapter.5}
\BOOKMARK [2][-]{subsection.5.3.1}{\376\377\000D\000e\000t\000a\000i\000l\000e\000d\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n}{section.5.3}
\BOOKMARK [1][-]{section.5.4}{\376\377\000m\000a\000i\000n\000T\000e\000s\000t\000.\000c\000c\000\040\000F\000i\000l\000e\000\040\000R\000e\000f\000e\000r\000e\000n\000c\000e}{chapter.5}
\BOOKMARK [2][-]{subsection.5.4.1}{\376\377\000D\000e\000t\000a\000i\000l\000e\000d\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n}{section.5.4}
\BOOKMARK [2][-]{subsection.5.4.2}{\376\377\000F\000u\000n\000c\000t\000i\000o\000n\000\040\000D\000o\000c\000u\000m\000e\000n\000t\000a\000t\000i\000o\000n}{section.5.4}
\BOOKMARK [3][-]{subsubsection.5.4.2.1}{\376\377\000m\000a\000i\000n}{subsection.5.4.2}
